params/history_segment_config.go

```diff
@@ -0,0 +1,56 @@
+package params
+
+const (
+	BoundStartBlock      uint64 = 31268600 // The starting block height of the first segment, was produced on Aug-29-2023
+	HistorySegmentLength uint64 = 2592000  // Assume 1 block for every 3 second, 2,592,000 blocks will be produced in 90 days.
```

Comment:
https://docs.google.com/document/d/1txmWxATz-VnZpH63SWAEsBVrEr6HMtNZtsK3ZtaV0mQ/edit
history block data has increased ~500GB in 2023.
After a second though, maybe we can set it to 8months or even 1year, then segment change will be much slower.